api/src/DataFixtures/ProfTestUserFixtures.php

```diff
@@ -543,7 +538,6 @@ private function addUser($data, $manager)
             ->setAddress3(isset($data['Dep Adrs3']) ? $data['Dep Adrs3'] : null)
             ->setAddressPostcode(isset($data['Dep Postcode']) ? $data['Dep Postcode'] : 'SW1')
             ->setAddressCountry('GB')
-            ->setDeputyNo(isset($data['Deputy No']) ? $data['Deputy No'] : null)
```

Comment:
Removing setting the DeputyNo on the User entity for Org cases fixtures.
This is only ever done for Lay cases.